### DISCUSSION TITLE
router alias relative location doesn't work

### DISCUSSION THREAD

**Discussion by lyqyq2001** [vuejs/router#2303] [Help and Questions] [answered]
![Snipaste_2024-07-12_18-14-10](https://github.com/user-attachments/assets/b0aeafc9-dfa5-48a2-8fb1-2a6be88f8296)

**Comment by lyqyq2001** [comment]
it  doesn't like vue docs teach me. relative location doesn't start with '/'

**Comment by lyqyq2001** [comment]
maybe the docs description is wrong. should be fix it.

**Comment by skirtles-code** [comment, marked as answer]
Which part of the docs are you looking at?

You can use a relative path with `alias`, but it is relative to the parent route, just like the `path`. It isn't relative to the `path` of the current route. In your example the route doesn't have a parent, so it can't use a relative path.

e.g.:

```js
{
  path: '/user',
  component: UserView,
  children: [
    {
      path: 'hello',
      alias: 'abc',
      component: ListView
    }
  ]
},
```

The `alias` above maps to `/user/abc`.

Applying that to your code, you could maybe write it something like this:

```js
{
  path: '/hello',
  children: [
    {
      path: '',
      alias: 'abc',
      component: () => import('../Hello.vue')
    }
  ]
},
```

If you don't have any other children then it might be simpler to skip the nesting and use `alias: '/hello/abc'` instead of a relative path.

**Reply by lyqyq2001** [reply]
thank you for the reply. now i undestand
